src/locales/en.json

```diff
@@ -178,6 +178,7 @@
     "risk_group_empty_warning": "Enter your location first.",
     "saved_scenario_loaded_message": "If needed, adjust the following prefilled answers to match your <strong>{{ scenarioName }}</strong> scenario.",
     "alerts": {
+      "vaccines_update_3": "<0>March 30th, 2021:</0> The calculator and Risk Tracker now include support for Johnson & Johnson's vaccine. Pfizer & Moderna's effectiveness have been updated. Learn more <2>here</2>.",
```

Comment:
Should we mention increasing the wait time to two weeks per dose?

Reply:
I think so. 👍

Reply:
Done.